### ISSUE TITLE
Few drawing operations

### ISSUE THREAD

**Issue by pathumego** [trufont/trufont#176]
@adrientetar Can you tell me what you think of these?

- [ ] Align Points (#277)
- [x] Make line 
- [ ] Double click on a countour to select the contour (#276)
- [x] Change path direction (27ec4b0)
- [x] Preview mode #165 (https://github.com/trufont/trufont/commit/bd8d032db31fffe0f8bd982246ea857b0224f022)
- [x] Snap control points vertically and horizontally with Shift + Drag 
- [x] Add Curves (alt+click on line)
- [ ] Chord keys for tools - (eg hold space for hand/panning tool, ctrl for select tool) (#275)

(edited with suggestions in comments)

**Comment by madig** [comment]
I'm reading https://glyphsapp.com/tutorials/sketching and now really want everything they show in that tutorial :B

Basic point manipulation:
* Align Selection
* Add Curves (as described in the tutorial, including deleting behavior)
* Move points without moving handles

Alignment tools:
* Adjust Curvatures pane
* Equalize Stems, maybe along with a full alignment palette similar to Inkscape and Glyphs (https://github.com/fontforge/fontforge/issues/283). This would have made my work on Cantarell MUCH easier. Gah!

Basically tools that make turning a sketch into a polished outline easier.

**Comment by pathumego** [comment]
@adrientetar Would you like these kind of requests to be split into different issues?

**Comment by adrientetar** [comment]
> Make line

What does this mean?

> Adjust Curvatures pane

I think Tunni lines is better suited as it's a generalization of this.

**Comment by pathumego** [comment]
@adrientetar Select two points and make the segment between them a line. Maybe a better solution would be to enable deletion of control points.

**Comment by adrientetar** [comment]
> deletion of control points.

Select offcurve and hit the delete key.

**Comment by madig** [comment]
I usually use "make line" when I'm cutting up some glyph to derive some other glyph from it, to connect loose ends.

**Comment by davelab6** [comment]
On 9 January 2016 at 12:49, madig <notifications@github.com> wrote:

> I usually use "make line" when I'm cutting up some glyph to derive some
> other glyph from it, to connect loose ends.
>
I think this operation is very common! :)

**Comment by adrientetar** [comment]
> Move points without moving handles

54772066bdc3552ad4089b1ca4f3d576777ce6e0

**Comment by madig** [comment]
@adrientetar Uh, how to use it? Nothing different happens when I shuffle a point around with Alt pressed and arrow keys. Handles move along as usual.

**Comment by adrientetar** [comment]
It's with the mouse.

**Comment by madig** [comment]
Ah, I got it, it only works for curve points, not corner points. Please also implement it for arrow keys and when selecting multiple points.

**Comment by pathumego** [comment]
@adrientetar if there is any way to up vote `Preview Mode`, this is me Up voting :hand: :D

**Comment by davelab6** [comment]
I think this is a bit premature. 

For those users who are eager to start, though, then I suggest accumulating "tips and tricks" for how **YOU** personally do the things that you frequently do. Screencasts are probably ideal for this, along with something that shows keypresses and mouseclicks on screen. 

For mac there are a couple of proprietary apps for that kind of thing, http://silverbackapp.com + https://www.boinx.com/mousepose/overview/ 

For windows and libre desktops, I'm not sure what the best stuff is. https://www.google.com/search?q=key-mon+github is a start... I am sad that @felipesanches never really shipped a libre version of  for GNOME3, and don't know what libre tools to recommend to make narrated screencasts (if not video-narrated as silverback offers) but I think an easy proprietary solution is Google Hangouts on Air.

**Comment by pathumego** [comment]
All the unresolved stuff moved to separate issues.

**Comment by davelab6** [comment]
@madig said offline,

> Align is a must, click-dragging curves (in addition to handles) is helpful and equalize stems and adjust curvatures are nice to have. Nudge is thankfully already there.
>
> The overarching goal with these is to produce smooth, kinkless curves and consistent stems, the daily battle of any font designer and therefore basic functionality :)
